compiler/dcalc/from_scopelang.ml

```diff
@@ -682,7 +693,7 @@ let translate_rule
     let thunked_or_nonempty_new_e =
       match a_io.Desugared.Ast.io_input with
       | Runtime.NoInput, _ -> assert false
-      | Runtime.OnlyInput, _ -> Expr.eerroronempty new_e (Mark.get new_e)
+      | Runtime.OnlyInput, _ -> new_e
```

Comment:
Wouahou I was putting a lot of errors on empty where it was not necessary...

Reply:
Yes, well it was very hard to ensure they were everything where needed without the extra typing :)